_posts/en/newsletters/2018-08-21-newsletter.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 'Bitcoin Optech Newsletter #9'
+permalink: /en/newsletters/2018/08/21/
+name: 2018-08-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes the usual dashboard and action items, 
+short descriptions of projects Bitcoin Core contributors are working on,
+and a list of notable merges during the past week.
+
+## Action items
+
+- **Allocate time to test Bitcoin Core 0.17 release candidates:**
+  in the coming days, Bitcoin Core will begin releasing Release
+  Candidates (RCs) for version 0.17.0.  Organizations and individual
+  users planning to use 0.17 are encouraged to test the RCs to ensure
+  that they contain all the features you need and don't have any bugs
+  that would affect your operation.  There are often multiple RCs for a
+  major release such as this, but each RC can theoretically be the last
+  RC, so we encourage you to test as early as possible.
+
+## Dashboard items
+
+<!-- TODO: @moneyball -->
+
+## News
+
+No significant news was posted to the bitcoin-dev or lightning-dev
+mailing list last week, so this week's news focuses on some projects
+discussed during the Bitcoin Core weekly meeting.
+
+The Bitcoin Core project, like most free and open source software
+projects, is organized bottom-up with each contributor working on the
+things they think are important rather than top-down with project
+leaders directing the work, so occasionally---as happened last
+week---developers briefly summarize to each other what they're working
+on for the future.  It's possible some of these initiatives may fail,
+but it's also possible that some will become future parts of Bitcoin
+Core.  Here's a summary of the projects discussed:
+
+- **P2P protocol encryption** being worked on by Jonas Schnelli with
+  a short-term focus on unauthenticated encryption in the [BIP151][]
+  style (but perhaps using a different mechanism than described in that
+  BIP).  Peer authentication (e.g. [BIP150][]) is probably further off
+  as a criticism against it is that the simplest way of implementing it
+  makes it easy to fingerprint particular peers and reduce privacy---so
+  a more advanced mechanism is desired for the cases that need it.
+
+- **Output script descriptors enhancements** being worked on by
+  Pieter Wuille.  The basic idea for this was described in
+  [Newsletter #5][news5 news] but Wuille is investigating adding
+  support for the ability to "import `and(xpub/...,or(xpub/...,xpub/...))`
```

Comment:
I think using the words "nested and/or/threshold constructions" as Pieter did in IRC is a clearer way to introduce this (but maybe that's just a matter of personal taste)